app/src/main/res/values/strings.xml

```diff
@@ -40,7 +40,7 @@
     <string name="gender">Gender*</string>
     <string name="height">Height*</string>
     <string name="weight">Weight*</string>
-    <string name="yearTooRecent">Invalid year : You can not be born in the future </string>
+    <string name="yearTooRecent">Invalid year : You have to be an adult (and cannot be born in the future) </string>
```

Comment:
If you want you can split this in two different error message

Reply:
It would have meant changing the create test field method :sweat_smile: